crates/bins/wasmflow/src/commands/run.rs

```diff
@@ -34,14 +37,52 @@ pub(crate) async fn handle_command(opts: RunCommand) -> Result<()> {
 
   debug!(args = ?opts.args, "rest args");
 
-  let bytes = wasmflow_loader::get_bytes(&opts.location, opts.fetch.allow_latest, &opts.fetch.insecure_registries)
-    .await
-    .context("Could not load from location")?;
+  let inherent_data = opts.seed.map(|seed| {
+    InherentData::new(
+      seed,
+      SystemTime::now()
+        .duration_since(SystemTime::UNIX_EPOCH)
+        .unwrap()
+        .as_millis()
+        .try_into()
+        .unwrap(),
+    )
+  });
+
+  let app_config = wasmflow_runtime::configuration::load_yaml(&opts.location)?;
+  let context = ApplicationContext {
+    name: app_config.name,
+    version: app_config.version,
+    inherent_data,
+  };
+
+  let mut channels: Vec<Box<dyn Channel + Send + Sync>> = vec![];
+  for (name, channel_config) in app_config.channels {
+    debug!("Loading channel {}", name);
+    match wasmflow_runtime::configuration::get_channel_loader(&channel_config.uses) {
+      Some(loader) => channels.push(loader(context.clone(), channel_config.with)?),
+      _ => bail!("could not find channel {}", &channel_config.uses),
+    };
+  }
 
-  if crate::wasm::is_wasm(&bytes) {
-    todo!("The run command is not yet enabled for wasm modules.");
-    // wasm::handle_command(opts, bytes).await
-  } else {
-    manifest::handle_command(opts, bytes).await
+  let mut tasks: Vec<JoinHandle<Result<(), anyhow::Error>>> = vec![];
+  for channel in channels {
```

Comment:
```suggestion
  for channel in &channels {
```
Could we just borrow channel as a reference here? Since Channel is inside a Box, maybe the move in the closure won't be an issue.

Reply:
It needed an `Arc` but it's fixed now. Thanks @wuerges